README.md

```diff
@@ -275,6 +250,31 @@ python3 -m pip install git+https://github.com/mozilla-iot/gateway-addon-python.g
     $ cd gateway
     ```
 
+* Use nvm to install and use an LTS version of node and then set the default version. The **`.nvmrc`** file will be used by nvm to determine which version of node to install.
+
+    ```
+    $ nvm install
+    $ nvm use
+    $ nvm alias default $(node -v)
+    ```
+
+* Verify that node and npm have been installed:
+
+    ```
+    $ node --version
+    v8.12.0
+    $ npm --version
+    6.4.1
+    ```
+
+    Note: these versions might differ from the LTS version installed locally.
+
+* Reinitialize your terminal session.
```

Comment:
Remove the `Reinitialize` part and keep it where it was.

Reply:
Done